app/assets/javascripts/events.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
This file can be deleted. Consider adding the following line to [here in `config/application.rb`](https://github.com/xudongcamsys/homefoodie/blob/master/config/application.rb#L21) to prevent empty assets being generated in future:

```ruby
  g.assets = false
```